modules/etcd-certificates.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="etcd-certificates_{context}"]
+= etcd certificates
+
+[discrete]
+== Purpose
+
+etcd certificates are signed by the etcd-signer; they come from a certificate
+authority (CA) that is generated by the bootstrap process.
+
+[discrete]
+== Location
+
+* CA certificates:
+** etcd CA certificate: `/etc/ssl/etcd/ca.crt`
+** etcd metric CA certificate: `/etc/ssl/etcd/metric-ca.crt`
+* Server certificates: `/etc/ssl/etcd/system:etcd-server`
+* Client certificates: `<api_server_pod_directory>/secrets/etcd-client/`
+* Peer certificates: `/etc/ssl/etcd/system:etcd-peer`
+* Metric certificates: `/etc/ssl/etcd/metric-signer`
```

Comment:
regarding to 'metric-signer', I can't find it in the right dir, pls help to correct me if any misunderstanding: 
sh-4.2# cd /etc/ssl/
sh-4.2# ls
certs  etcd
sh-4.2# cd etcd/
sh-4.2# ls
ca.crt									  system:etcd-peer:etcd-0.xxx.qe.gcp.devcluster.openshift.com.crt
metric-ca.crt								  system:etcd-peer:etcd-0.xx.qe.gcp.devcluster.openshift.com.key
root-ca.crt								  system:etcd-server:etcd-0.xx.qe.gcp.devcluster.openshift.com.crt
system:etcd-metric:etcd-0.xxx.qe.gcp.devcluster.openshift.com.crt  system:etcd-server:etcd-0.xxx.qe.gcp.devcluster.openshift.com.key
system:etcd-metric:etcd-0.xxx.qe.gcp.devcluster.openshift.com.key
sh-4.2#

Reply:
@hexfusion Can you please review?

cc @pweil-  @deads2k